File: SofiForce.Models/Models/ListModels/ClientServiceRequestTimlineListModel.cs
using System;

namespace Models
{
    public class ClientServiceRequestTimlineListModel
    {
        public string Id { get; set; }
        public long TimelineId { get; set; }
        public long? RequestId { get; set; }
        public int? RequestStatusId { get; set; }
        public int? UserId { get; set; }
        public DateTime? TimelineDate { get; set; }
        public string Notes { get; set; }
        public string RequestStatusName{ get; set; }
        public string RealName { get; set; }
        public string Color { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/ListModels/ClientServiceRequestTimlineListModel.ts
export interface ClientServiceRequestTimlineListModel {
    timelineId: number;
    requestId: number | null;
    requestStatusId: number | null;
    userId: number | null;
    timelineDate: Date | null;
    notes: string;
    requestStatusName: string;
    realName: string;
    color: string;
}